t/t2608-job-shell-log.t

```diff
@@ -133,12 +133,20 @@ for topic in "shell.init" "shell.exit" \
 done
 
 test_expect_success 'flux-shell: missing command logs fatal error' '
-	test_expect_code 1 flux mini run nosuchcommand 2>missing.err &&
+	test_expect_code 127 flux mini run nosuchcommand 2>missing.err &&
 	grep "flux-shell\[0\]: FATAL: task 0: start failed" missing.err &&
 	grep "job.exception type=exec severity=0 task 0: start failed" missing.err &&
         grep "No such file or directory" missing.err
 '
 
+test_expect_success 'flux-shell: illegal command logs fatal error' '
+        mkdir adirectory &&
+	test_expect_code 126 flux mini run adirectory 2>illegal.err &&
+	grep "flux-shell\[0\]: FATAL: task 0: start failed" illegal.err &&
+	grep "job.exception type=exec severity=0 task 0: start failed" illegal.err &&
+        grep "Permission denied" illegal.err
```

Comment:
nit: looks like maybe a couple indentation errors in this test?

Reply:
Yup, there was a tab vs space issue.  Went through the file and it seems we got a lot of tab vs space inconsistencies, and in other tests.  Maybe should do a big cleanup one of these days.